Documentation/compatibility/Stack-traces-obtained-when-using-portable-PDBs-now-include-source-file-and-line-information-if-requested.md

```diff
@@ -0,0 +1,51 @@
+## Stack traces obtained when using portable PDBs now include source file and line information if requested
+   
+### Scope
+Edge
+
+### Version Introduced
+4.7.2
+
+### Source Analyzer Status
+NotPlanned
+
+### Change Description
+Stack traces obtained when using portable PDBs will now include source file and line information when requested. In versions prior to 4.7.2 source file and line information would be unavailable when using portable PDBs even if explicitly requested.
```

Comment:
- Best to begin with, "Starting with .NET Framework 4.7.2,, stack traces..."
- will now include --> include
- 4.7.2 source file --> .NET Framework 4.7.2, source file
would be --> was